can_tools/scrapers/official/VT/vt_vaccinations.py

```diff
@@ -1,15 +1,11 @@
+from typing import Dict
```

Comment:
Looks like Vermont restructured the underlying data for their dashboard. Unfortunately, I couldn't find raw cumulative counts anywhere in the data, but they did provide %s and population values, so I use that to calculate cumulative counts.